net/lldpd/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+PortGroup             github 1.0
+
+github.setup          lldpd lldpd 1.0.19
+revision              0
+categories            net
+maintainers           {@akkornel stanford.edu:akkornel} openmaintainer
+license               ISC
+
+description           LLDP allows you to know exactly on which port is a \
+                      machine (and reciprocally).
+long_description      lldpd is an 802.1ab implementation (LLDP) to help you \
+                      locate neighbors of all your equipments.  Proprietary \
+                      protocols (like EDP and CDP) are supported. \
```

Comment:
This line continuation (`\`) will interfere with the `homepage` line.

Reply:
Yup, you are correct, thanks for being patient with me!